src/components/pagination/PaginationSubBox.js

```diff
@@ -12,191 +12,250 @@ import CheckBlueImage from '../../images/icon-check-blue.svg'
 import styled from 'styled-components'
 
 const CustomList = styled(List)`
-   &&& {
-      background: #fff;
-      width: 100%;
-      height: 100%;
-      padding: 0 0;
-
-      > .item {
-         padding: 18px 20px;
-      }
-      .img {
-         width: 20px;
-         position: absolute;
-         top: 20px;
-         right: 20px;
-         padding: 0px;
-
-         img {
-            cursor: pointer;
-         }
-      }
-      .text {
-         margin: 0 10% 0 0;
-         color: #24272a;
-         float: left;
-
-         .header {
-            font-family: 'benton-sans', sans-serif;
-         }
-         .content {
-            color: #999999;
-            padding-top: 12px;
-            line-height: 20px;
-         }
-      }
-      .remove-connection {
-         > button {
-            width: 100%;
-            background-color: #ff585d;
-            border: 2px solid #ff585d;
-            border-radius: 25px;
-            color: #fff;
-            font-weight: 600;
-            margin-top: 12px;
-
-            :hover {
-               background: #fff;
-               color: #ff585d;
-            }
-         }
-      }
-      .recent-transactions {
-         background-color: #f8f8f8;
-         margin: 0 0 0 0;
```

Comment:
Minor: `margin: 0` should suffice. There are a few occurrences. Same for `padding` cases.